xdsl/transforms/experimental/ConvertStencilToLLMLIR.py

```diff
@@ -359,10 +360,53 @@ def map_returns(op: Operation) -> None:
     return return_targets
 
 
+def all_matching_uses(op_res: Iterable[SSAValue], typ: type[_TypeElement]) -> _TypeElement | None:
```

Comment:
```suggestion
def all_matching_uses(op_res: Iterable[SSAValue], typ: type[_TypeElement]) -> Generator[_TypeElement, None, None]:
```

Reply:
But also `_TypeElement` is `bound=Attribute`, you want to use a different value, probably `OperationInvT`